types/rpi-gpio/rpi-gpio-tests.ts

```diff
@@ -0,0 +1,36 @@
+import gpio from "rpi-gpio";
```

Comment:
Please use the commonjs syntax `import gpio = require('rpi-gpio')`, and remove "esModuleInterop": true from tsconfig.

Consumers of the package who want to use `import gpio from 'rpi-gpio'` will need to add `"esModuleInterop": true`, but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases.